server/controllers/notification.js

```diff
@@ -0,0 +1,50 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+// createNotification, readNotification, getAll, getUnread
+exports.createNotification = asyncHandler(async (req, res) => {
+    const notification = JSON.parse(req.body)
```

Comment:
Please destructure the request body to only allow required fields for increased security

Reply:
Changed in commit.